### DISCUSSION TITLE
Telegram integration supergroup

### DISCUSSION THREAD

**Discussion by Amonlol** [grafana/grafana#78141] [General]
Hello, loving your product! 

Is it possible to send alert messages through telegram bot into a specific topic inside supergroup? It is achieved with setting ``message_thread_id`` parameter in url request like this: ``https://api.telegram.org/<my_API_key>/sendMessage?chat_id=-100<chat_id>&message_thread_id=<topic_id>``

Tried to set both chat_id and message_thread_id ``chatid`` like this: `"chatid": "-100<chat_id>&message_thread_id=<topic_id>"`, but no luck - it still sends messages into the general topic...

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues. You can read more about the move https://github.com/grafana/grafana/discussions/73424.

This one in particular sounds like it may be a better fit for our [community forum](https://community.grafana.com/), feel free to create a post there. Sorry for the inconvenience, and thanks for your contributions!
